### ISSUE TITLE
S-2019-002-JG: Recordkeeping

### ISSUE THREAD

**Issue by rachelduggan10** [dlounsbu/PRIME_studios#27]
- [ ] Enter new experts in stakeholder data base.
- [ ] Process evaluation surveys.
- [ ] Update SE Studio record data database.

**Comment by dlounsbu** [comment]
@claudialec @rachelduggan10 

As I think about it, there are really just two RedCap instruments to develop for the Participants: 
1. Stakeholder Recruitment Database/Survey 
2. Stakeholder Participant Studio Evaluation Survey

And the only other Redcap instrument to design would be the requesting investigator, and I would all this instrument:
3. Stakeholder Engagement Studio Request Form

DL

**Comment by claudialec** [comment]
Correct.  Thats essentially whats in there now, but we should discuss over applicability and intended use beyond this.    It may be easier if I create a flow diagram to help visualize layout and content. I'll aim for that for next week's meeting.
